components/Header.js

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+
+export default function Header() {
+  const [isScrolled, setIsScrolled] = useState(false);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      if (window.scrollY > 20) {
+        setIsScrolled(true);
+      } else {
+        setIsScrolled(false);
+      }
+    };
+
+    window.addEventListener("scroll", handleScroll);
+
+    return () => {
+      window.removeEventListener("scroll", handleScroll);
+    };
+  }, []);
+
+  return <StyledHeader>Money Manager</StyledHeader>;
```

Comment:
If you like, you could add an invisible `Link` to the homepage to the header in the future. I find this intuitive and practical when you have multiple pages.